src/main/java/cz/xtf/XTFConfiguration.java

```diff
@@ -0,0 +1,744 @@
+	private static final String PROXY_DOMAIN = "xtf.config.domain.proxy";
+	private static final String SOAK_TEST_ITERATIONS = "xtf.config.soak_test_iterations";
+	private static final String MAVEN_BASE_URL = "xtf.config.maven.web.url";
+	private static final String MAVEN_PROXY_URL = "xtf.config.maven.proxy.url";
+	private static final String MAVEN_PROXY_GROUP = "xtf.config.maven.proxy.group";
+	private static final String MAVEN_PROXY_ENABLED = "xtf.config.maven.proxy.enabled";
+	private static final String MAVEN_DEPLOY_SNAPSHOT_URL = "xtf.config.maven.deploy.snapshot.url";
+	private static final String MAVEN_DEPLOY_RELEASE_URL = "xtf.config.maven.deploy.release.url";
+	private static final String GITLAB_URL = "xtf.config.gitlab.url";
+	private static final String GITLAB_TOKEN = "xtf.config.gitlab.token";
+	private static final String GITLAB_USERNAME = "xtf.config.gitlab.username";
+	private static final String GITLAB_PASSWORD = "xtf.config.gitlab.password";
+	private static final String GITLAB_GROUP = "xtf.config.gitlab.group";
+	private static final String GITLAB_GROUP_ENABLED = "xtf.config.gitlab.group.enabled";
+	private static final String PING_PROTOCOL = "xtf.config.ping.protocol";
+	private static final String FABRIC8_VERSION = "xtf.config.fabric8.version";
+	private static final String FABRIC8_SERVICE_NAME_LIMIT = "xtf.config.fabric8.service_name_limit";
+	private static final String FUSE_CACHED_IMAGES = "xtf.config.fuse.cached.images";
+	private static final String FUSE_DISABLE_JOLOKIA = "xtf.config.fuse.disable.jolokia";
+	private static final String BUILD_NAMESPACE = "xtf.config.build.namespace";
+	private static final String FORCE_REBUILD = "xtf.config.build.force.rebuild";
+	private static final String BINARY_BUILD = "xtf.config.build.binary";
+	private static final String MAX_HTTP_TRIES = "util.http.maxtries";
+	private static final String DEFAULT_WAIT_TIMEOUT = "xtf.config.wait.timeout.default";
+	private static final String EXTERNAL_SERVICES_HOST = "xtf.config.services.external";
+	private static final String OPENSHIFT_ONLINE = "xtf.config.openshift.online";
+	private static final String OPENSHIFT_VERSION = "xtf.config.openshift.version";
+	private static final String TEST_ALERT = "test.alert";
+	private static final String TEST_JENKINS_RERUN = "test.jenkins.rerun";
+	private static final String TEMPLATE_REPO = "xtf.config.template.repo";
+	private static final String TEMPLATE_BRANCH = "xtf.config.template.branch";
+
+	private static final String OPENSTACK_URL = "xtf.config.openstack.url";
+	private static final String OPENSTACK_TENANT = "xtf.config.openstack.tenant";
+	private static final String OPENSTACK_OPEN_SECURITY_GROUP = "xtf.config.openstack.security.group.open";
+	private static final String OPENSTACK_USERNAME = "xtf.config.openstack.username";
+	private static final String OPENSTACK_PASSWORD = "xtf.config.openstack.password";
+
+
+	public static final String OC_BINARY_LOCATION = "oc.binary.location";
```

Comment:
Why did this property moved to XTFConfiguration while it's static method stood in TestConfiguration?

Reply:
XTFConfiguration should be like a base configuration class.
So I think it is a common attribute.

Reply:
Shouldn't then also method TestConfiguration.ocBinaryLocation() be rather in XTFConfiguration where it's property lies? Properties and methods should lie within same file.

https://github.com/xtf-cz/xtf/pull/55/files#diff-3cc59395e886593785f8f1ac99a1ba5cR73